Source/player.cpp

```diff
@@ -1890,18 +1759,19 @@ int Player::GetCurrentAttributeValue(CharacterAttribute attribute) const
 
 int Player::GetMaximumAttributeValue(CharacterAttribute attribute) const
 {
-	static const int MaxStats[enum_size<HeroClass>::value][enum_size<CharacterAttribute>::value] = {
-		// clang-format off
-		{ 250,  50,  60, 100 },
-		{  55,  70, 250,  80 },
-		{  45, 250,  85,  80 },
-		{ 150,  80, 150,  80 },
-		{ 120, 120, 120, 100 },
-		{ 255,   0,  55, 150 },
-		// clang-format on
-	};
-
-	return MaxStats[static_cast<std::size_t>(_pClass)][static_cast<std::size_t>(attribute)];
+	PlayerData plrData = PlayersData[static_cast<std::size_t>(_pClass)];
+	switch (attribute) {
+	case CharacterAttribute::Strength:
+		return plrData.maxStr;
+	case CharacterAttribute::Magic:
+		return plrData.maxMag;
+	case CharacterAttribute::Dexterity:
+		return plrData.maxDex;
+	case CharacterAttribute::Vitality:
+		return plrData.maxVit;
+	default:

```

Comment:
Best not have a `default` branch in enum switches.
Without the default branch, if a new value is added to the enum and is not handled, the compiler will warn you.
Can put `app_fatal` after the `switch`